src/sections/node/channels.tsx

```diff
@@ -39,6 +48,8 @@ function ChannelsPage() {
       }
     >
   >({});
+  const tabLabel = tabIndex === 0 ? 'outgoing' : 'incoming';
+  const channelsData = tabIndex === 0 ? channels?.incoming : channels?.outgoing;
```

Comment:
The logic here is switched, resulting in not being able to export the "incoming" channels when I have 22 open, but being able to export the "outgoing" which I have non. 
```suggestion
  const tabLabel = tabIndex === 0 ? 'outgoing' : 'incoming';
  const channelsData = tabIndex === 0 ? channels?.outgoing : channels?.incoming;
```

small demo of what I mean:
![Download Channels CSV bug](https://github.com/hoprnet/hopr-admin/assets/32660456/8e65aa00-a989-42ae-92d3-d22e5b57fbfb)